dev/conductor/ui/pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   string_scanner: 1.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   term_glyph: 1.2.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   typed_data: 1.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  desktop_window: ^0.4.0
```

Comment:
sorry, I should have mentioned that you should try to avoid adding any pub dependencies. since the flutter/flutter repository is a monorepo, any version constraint in one dart package constrains all of them. this could make upgrading pub dependencies more difficult.

Also, since this repository is not owned by a Google team, we may need a security review. Are you only using this to set the initial windows size?

Reply:
yes, I am only using it to set the window size. Do you know a way without using this package?

Reply:
Not in Dart, I suspect you would need to use native code to do this. If we just want to set the initial window size, maybe you could just add it to the linux and macos platform code?

Reply:
Also note Casey's comment--can we not just rely on the default size?

Reply:
we could, but the user could minimize the window to a really small size, and causing some overflow errors.

Reply:
https://user-images.githubusercontent.com/20194490/135176696-d08ccf34-87f5-446c-9f58-7b35f9aeeb70.mov

Reply:
Let's ignore this for now, and revisit if we get feedback on it